pyproject.toml

```diff
@@ -1,12 +1,13 @@
 [tool.poetry]
 name = "shipchain-common"
-version = "0.1.0"
+version = "0.1.3"
```

Comment:
Let's change this back to 0.1.0 for publishing on PyPI - I know TestPyPI won't let you overwrite versions so you had to increment this for testing, but I'd like to release a clean 0.1.0 version on PyPI.